fsspec/registry.py

```diff
@@ -251,3 +251,8 @@ def filesystem(protocol, **storage_options):
     """
     cls = get_filesystem_class(protocol)
     return cls(**storage_options)
+
+
+def available_protocol():
+    """Return a list of the available protocols."""
```

Comment:
Can you please make this more specific: these are protocols that fsspec knows how to deal with, but they still might not be "available" if other required dependencies are missing.

Reply:
> Do you mean to correct the docstring

Yes, exactly. Doing a load of imports is probably a bad idea.

> I think would be useful to add it to the API docs. Should I do a PR also for that?

You can include here - see  docs/source/api.rst .

Reply:
Hey @martindurant. Doc updated. 
Note that I also added to the registry an additional protocol "local" as a proxy for "LocalSystem" (in addition to the existing "file").

Reply:
> I also added to the registry an additional protocol "local"

That's fine